woo-tracking-jsdoc/publish.js

```diff
@@ -0,0 +1,77 @@
+const fs = require( 'jsdoc/fs' );
+const env = require( 'jsdoc/env' );
+const path = require( 'jsdoc/path' );
+
+// RegExp used to match the replacement. The groups are respectively: start marker, replacable content, end marker.
+const defaultReplacementRegex = /(<woo-tracking-jsdoc.*>)([\s\S]*)(<\/woo-tracking-jsdoc.*>)/;
+
+const disclaimer = `<!---
+Everything inside \`<woo-tracking-jsdoc>\` will be automatically generated by \`woo-tracking-jsdoc\`.
```

Comment:
💅 
How about filling the matched tag here?

```suggestion
Everything inside \`$1\` will be automatically generated by \`woo-tracking-jsdoc\`.
```

Another idea is to add comments at the beginning and end of the auto-generated blocks, but without the tag name. So that when multiple lines of tag and its attributes are matched, there would be no formatting problems when multiple lines of content are placed inside the \`\`.

Reply:
> Another idea is to add comments at the beginning and end of the auto-generated blocks, but without the tag name.

Great idea thanks. Implemented in https://github.com/woocommerce/pinterest-for-woocommerce/pull/238/commits/252faa6a820a9d9b22d3cda868ce03cacb791f25